aten/src/ATen/native/mps/operations/Activation.mm

```diff
@@ -959,16 +959,23 @@ Tensor relu_mps(const Tensor& self) {
 }
 
 void elu_variants_out_mps (
-  const Tensor& self,
+  const Tensor& self_,
   const Scalar& alpha,
   const Scalar& scale,
   const Scalar& input_scale,
-  const Tensor& result,
+  const Tensor& result_,
   string func_name) {
 
   using namespace mps;
-  TORCH_CHECK(self.is_mps());
+  TORCH_CHECK(self_.is_mps());
```

Comment:
Do we need this check anymore? We have this same check in our Placeholder(). Is it ever possible that we reach this code, and self wouldn't be an MPS tensor? The PyTorch dispatchers shouldn't call into MPS functions if tensors aren't on MPS device.

Reply:
Thanks. Updated